adafruit_stmpe610.py

```diff
@@ -368,7 +379,7 @@ def touch_point(self):  # pylint: disable=too-many-branches
             return (x, y, pressure)
         return None
 
-    def _read_register(self, register, length):
+    def _read_register(self, register: int, length: int) -> List[int]:
```

Comment:
While technically `bytearray` **is** a list of `int`, we can be more specific and type it as `bytearray` return.